datasource-remote/src/main/java/cmm/apps/esmorga/datasource_remote/api/EsmorgaApi.kt

```diff
@@ -13,4 +14,7 @@ interface EsmorgaApi {
 
     @POST("account/events")
     suspend fun joinEvent(@Body body: Map<String, String>)
+
+    @HTTP(method = "DELETE", path = "account/events", hasBody = true)
```

Comment:
Why are you using @HTTP instead of @DELETE?

Reply:
because there is an issue in retrofit with delete with body, and this is the workaround

Reply:
then I'd suggest to add a comment with the link to the issue so this can be changed when fixed